GOVERNANCE.md

```diff
@@ -9,9 +9,21 @@ The [nodejs/node](https://github.com/nodejs/node) GitHub repository is
 maintained by Collaborators who are added by the CTC on an ongoing basis.
 
 Individuals identified by the CTC as making significant and valuable
```

Comment:
This is the original language and it is definitely lacking. Specific suggestions for improvement are welcome as I'm not sure what would be better